net-misc/cgminer/cgminer-4.11.1.ebuild

```
@@ -1,20 +1,21 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit flag-o-matic toolchain-funcs udev
 
 DESCRIPTION="Bitcoin CPU/GPU/FPGA/ASIC miner in C"
 HOMEPAGE="https://bitcointalk.org/?topic=28402.msg357369 https://github.com/ckolivas/cgminer"
 SRC_URI="http://ck.kolivas.org/apps/cgminer/${P}.tar.bz2"
-#SRC_URI="http://ck.kolivas.org/apps/cgminer/4.5/${P}.tar.bz2"
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~x86"
+KEYWORDS="~amd64 ~arm ~x86" # upstream claims mips compatibility
```

Comments like this aren't really helpful, we're not going to keyword something because upstream claims it works. We keyword something because there's a demand.
```suggestion
KEYWORDS="~amd64 ~arm ~x86"
```